mod_reforged/hooks/skills/actives/shieldwall.nut

```diff
@@ -18,6 +18,13 @@
 			text = ::Reforged.Mod.Tooltips.parseString("Grants immunity against [Hook Shield|Skill+rf_hook_shield_skill]")
 		});
 
+		tooltip.push({
+			id = 6,
```

Comment:
should have a unique id to improve submodding. Anything from 7 onwards is unused.